### PR TITLE
[FEAT] Add "select several BPMN Html Elements" example

### LINKED ISSUE #123: [FEAT] Add example about "select several BPMN Html Elements for interaction"
This is to create an example about https://github.com/process-analytics/bpmn-visualization-js/issues/927

### Proposal
https://github.com/process-analytics/bpmn-visualization-js/pull/954 provides a page that display info in a textarea which is not user friendly
We could display a toast when clicking (or mouse enter) on a element targeted by the BPMN kind using different colors depending on the kind. The toast could display the id and name.
We could also display the number of found elements
BPMN source: provide a list of bpmn example we have in this project and/or let user load their own from local device. This will demonstrate that we are not faking and this applies to any sources.

We could also add a specific behaviour for call activity: open an alert. This will complement the examples we already have for call activities.

### Implementation

Display toast on click, pool display/hide.
See #124

### PR DESCRIPTION
**Pull request by tbouffard**
closes #123 

**Live environment of examples**

https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/123-add_example_with_select_bpmn_elements_by_kind/examples/index.html

### CODE REVIEW

**Comment by tbouffard** [comment]
Here are some screencasts done during developments. The final implementation is closed to what they show.

select_end_event_v1_01_name_or_id

select_end_event_v1_02_also_in_subprocesses

select_pools_collapse_others_v1

select_pools_hide_others_v1

File: examples/custom-interaction/select-elements-by-bpmn-kind/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+In the following, we retrieve all `Pools` and `End Events`.
+
+```javascript
+const bpmnElements = bpmnVisualization.bpmnElementsRegistry.getElementsByKinds(
+    [ShapeBpmnElementKind.EVENT_END, ShapeBpmnElementKind.POOL]
+);
+```
+
+The returned array contains elements for both BPMN kinds.
+If you want to process the BPMN Kinds differently, you can filter the array as in the following if you first
```
Comment (csouchet) [inline examples/custom-interaction/select-elements-by-bpmn-kind/README.md:23]: It's not important, but everywhere is a 'k' and not 'K':

```suggestion
If you want to process the BPMN kinds differently, you can filter the array as in the following if you first
```

File: examples/custom-interaction/select-elements-by-bpmn-kind/README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+...
+```
+
+
+
+## Example implementation notes
+
+This example displays toast on `End Events` click thanks to [Notyf](https://github.com/caroso1222/notyf).
+
+It also allows hiding or collapsing `Pools`. The `bpmn-visualization` API is used to retrieve to the `Pool` ids but the `Pool`
```
Comment (csouchet) [inline examples/custom-interaction/select-elements-by-bpmn-kind/README.md:54]:
```suggestion
It also allows hiding or collapsing `Pools`. The `bpmn-visualization` API is used to retrieve to the `Pool` ids, but the `Pool`
```

File: examples/custom-interaction/select-elements-by-bpmn-kind/README.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+display if then fully managed by custom mxGraph code. \
```
Comment (csouchet) [inline examples/custom-interaction/select-elements-by-bpmn-kind/README.md:55]: Do you mean ?

```suggestion
display is then fully managed by custom mxGraph code. \
```

**Comment by csouchet** [comment]
## On MacOs

### Firefox
image
### Chrome
image
### Safari
image

**Review by csouchet — CHANGES_REQUESTED** [review]
There is a problem of rendering/display:
- No data in the _Select_
- The BPMN Container is too far to the left

**Comment by tbouffard** [comment]
> There is a problem of rendering/display:
> 
>     * No data in the _Select_
> 
>     * The BPMN Container is too far to the left

**No data in the Select**: the issue only occurs when accessing the example with the CDN url. It returns a wrong MIME type for the lib js file
https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/123-add_example_with_select_bpmn_elements_by_kind/examples/__tmp/bpmn-visualization@5be173e4.js: MIME type `text/html`
Actually, we get a 404 error when requesting this resource and in that case, statically.io serves a 404 html page, so this is why we got the html MIME type.
I have reproduced the issue on Ubuntu 18.04 with Chrome and Firefox

This js file has been added to the branch after the CDN cache has been filled and the cache is then not updated. See https://statically.io/docs/using-staticzap/
> All files will be cached on the CDN for 1 year, except for these listed branch that are cached for only 1 day:
>
>     main
>     master
>     dev
>     develop
>     gh-pages



**The BPMN Container is too far to the left**: consequence of the Javascript loading issue

**Review by csouchet — APPROVED** [review]

**Pull request merged**